hack/copy_binaries.sh

```diff
@@ -18,9 +18,13 @@
 # PERFORMANCE OF THIS SOFTWARE.
 
 BIN_PATH=/usr/local/bin/
+BIN_PATH2=/opt/runnc/bin/
```

Comment:
Why is this path needed?

Reply:
since `runnc` is the only thing that the user is going to use, i was moving `runnc-cont` and `nabla-run` into a directory where the user would not be able to directly call it. As discussed, will add this as a comment.